utilities/backupable/backupable_db.cc

```diff
@@ -928,13 +928,15 @@ BackupEngineImpl::BackupEngineImpl(const BackupEngineOptions& options,
       read_only_(read_only) {
   if (options_.backup_rate_limiter == nullptr &&
       options_.backup_rate_limit > 0) {
-    options_.backup_rate_limiter.reset(
-        NewGenericRateLimiter(options_.backup_rate_limit));
+    options_.backup_rate_limiter.reset(NewGenericRateLimiter(
+        options_.backup_rate_limit, 100 * 1000 /* refill_period_us */,
+        10 /* fairness */, RateLimiter::Mode::kAllIo /* mode */));
```

Comment:
I think this is an unnecessarily disruptive change in behavior, because there's an inconsistency in default behavior with NewGenericRateLimiter. Specifically, NewGenericRateLimiter only limits writes by default, whereas here we are changing the default for backup+restore to limit both reads and writes.

I would prefer either:
(a - easy and effective) Leave existing options as they are but comment on `BackupEngineOptions::backup_rate_limit` that the limit only applies to writes, and you must set `backup_rate_limiter` to have an option to also limit reads.
(b) Add option `bool backup_rate_limit_reads` to enable limiting reads under the limit given by `backup_rate_limit`

Reply:
Thanks for the suggestion - I will go with (a) for both backup and restore